_source/_posts/2018-05-22-simple-multifactor-authentication-in-node.md

```diff
@@ -65,23 +74,21 @@ npm install -g uuid-cli
 echo "APP_SECRET=`uuid`" >> .env
 ```
 
-For the next step, you need to [sign up for a free Okta Developer account](https://developer.okta.com/signup/), if you haven't already. Once you sign up, you'll be given a unique Okta Org URL. You'll need to save this in your `.env` file. Each line should have a separate environment variable with syntax like `ORG_URL=https://{yourOktaDomain}`.
+{% include setup/cli.md type="web" loginRedirectUri="http://localhost:8080/authorization-code/callback" logoutRedirectUri="http://localhost:8080/" %}
 
-After logging into your dev console, click the **Applications** tab, click **Add Application**, then select the **Web** option. You can keep all these settings at their default for now, and just change the name. If you already know the URI(s) where you will be hosting your application, this is where you would add them. You can always come back here later.
+Once you create the application, you will get **OKTA_OAUTH2_ISSUER**, **OKTA_OAUTH2_CLIENT_ID**, and **OKTA_OAUTH2_CLIENT_SECRET**. Update these in your `.env` file.
 
-{% img blog/simple-multifactor-authentication-in-node/add-application.png alt:"Add Application" width:"800" %}{: .center-image }
-
-Once you create the application, you will get a **Client ID** and **Client Secret**. You'll save these variables along with the **Base URI** you just entered in the last screen. Your `.env` file should now look something like this:
+Your `.env` file should now look something like this:
 
 ```bash
 APP_SECRET=super-secret-string
-ORG_URL=https://{yourOktaDomain}
+OKTA_OAUTH2_ISSUER=your-issuer-url
 HOST_URL=http://localhost:8080
-CLIENT_ID=your-client-id
-CLIENT_SECRET=your-client-secret
+OKTA_OAUTH2_CLIENT_ID=your-client-id
+OKTA_OAUTH2_CLIENT_SECRET=your-client-secret
 ```
 
-By keeping these variables in a separate file, it not only keeps them private, but also makes it easy to deploy the same code to different environments (e.g. development, staging, and production) by just swapping out one file.
+Keeping these variables in a separate file keeps them private. It makes it easy to deploy the same code to different environments (for example, development, staging, and production) by swapping out one file.
```

Comment:
You might want to re-evaluate this sentence. Maybe
> Keeping these variables in a separate file that is ignored by git keeps them private. 
?? not sure that's great but maybe add a reason why?